src/js/pinny.js

```diff
@@ -100,49 +104,58 @@
          */
         animation: {
             openComplete: function() {
-                var plugin = this;
-                this._disableExternalInputs();
-                this._focus();
+                setTimeout(function () {
+
+                    this._disableExternalInputs();
+                    this._focus();
+                    // only run lockup if another pinny isn't
+                    // open and locked the viewport up already
+                    if (this._activePinnies()) {
+                        this.$pinny.data('lockup')._trigger('locked');
+                    } else {
+                        this.$pinny.lockup('lock');
+                    }
 
-                // only run lockup if another pinny isn't
-                // open and locked the viewport up already
-                !this._activePinnies() && this.$pinny.lockup('lock');
+                    EventPolyfill.on(events.resize, this._repaint);
 
-                EventPolyfill.on(events.resize, function() {
-                    plugin._repaint();
-                });
+                    if (!this._activePinnies()) {
```

Comment:
Can this be combined into the `else` block above? [L115](https://github.com/mobify/pinny/pull/84/files#diff-ffb00137de8097bb2cd5cdddd1a8c432R115)